src/platform/graphics/graphics-device.js

```diff
@@ -140,7 +140,9 @@ class GraphicsDevice extends EventHandler {
         this._width = 0;
         this._height = 0;
 
-        this._maxPixelRatio = 1;
+        // Some devices window.devicePixelRatio can be less than one
+        // eg Oculus Quest 1
+        this._maxPixelRatio = Math.min(1, platform.browser ? window.devicePixelRatio : 1);
```

Comment:
If I understand the intent here, we don't want to allow ratio less than one. In that case we should use max instead of min?

Reply:
Ah, maybe I should explain in the comment better. We *do* want to allow the pixel ratio to be less than 1 otherwise we are rendering at higher resolution than the browser's native resolution.

In this case, the Quest DPR is 0.8 so before this change, the resolution of the canvas is higher with device pixel ratio disabled than it is with it enabled. As with it disabled, `_maxPixelRatio` and with it enabled, it is set at `window. devicePixelRatio` which would be 0.8

Reply:
```
this._maxPixelRatio = platform.browser ? Math.min(1, window.devicePixelRatio) : 1;
```

Reply:
Updated code on suggestion